tests/test_eval_detection.py

```diff
@@ -0,0 +1,95 @@
+# <Copyright 2020, Argo AI, LLC. Released under the MIT license.>
+"""Detection evaluation unit tests"""
+
+import logging
+import pathlib
+
+import numpy as np
+import pytest
+from pandas.core.frame import DataFrame
+from scipy.spatial.transform import Rotation as R
+
+from argoverse.data_loading.object_label_record import ObjectLabelRecord
+from argoverse.evaluation.detection_utils import DistFnType, SimFnType, dist_fn, sim_fn
+from argoverse.evaluation.eval_detection import DetectionCfg, DetectionEvaluator
+
+TEST_DATA_LOC = pathlib.Path(__file__).parent.parent / "tests" / "test_data" / "detection"
+logging.getLogger("matplotlib.font_manager").disabled = True
+
+
+@pytest.fixture
+def evaluator() -> DetectionEvaluator:
+    detection_cfg = DetectionCfg(significant_digits=32)
+    return DetectionEvaluator(
+        TEST_DATA_LOC / "detections", TEST_DATA_LOC, TEST_DATA_LOC / "test_figures", detection_cfg
+    )
+
+
+@pytest.fixture
+def metrics(evaluator: DetectionEvaluator) -> DataFrame:
+    return evaluator.evaluate()
+
+
+def test_center_similarity() -> None:
+    olr1 = np.array([ObjectLabelRecord(np.array([0, 0, 0, 0]), np.array([0, 0, 0]), 5.0, 5.0, 5.0, 0)])
+    olr2 = np.array([ObjectLabelRecord(np.array([0, 0, 0, 0]), np.array([3, 4, 0]), 5.0, 5.0, 5.0, 0)])
+    assert sim_fn(olr1, olr2, SimFnType.CENTER) == -5
+
+
+def test_iou_2d_similarity() -> None:
+    # TO DO
+    pass
+
+
+def test_iou_3d_similarity() -> None:
+    # TO DO
```

Comment:
are we planning to implement these later?

Reply:
Removing for now.